neofetch

```diff
@@ -1644,6 +1644,43 @@ get_term_font() {
             term_font="$(osascript -e 'tell application "Terminal" to font name of window frontmost')"
         ;;
 
+        "iTerm2")
+            # Unfortunately the profile name is not unique, but it seems to be the only thing
+            # that identifies an active profile. There is the "id of current session of current window"
+            # thou, but that does not match to a guid in the plist.
+            # So, be warned! Collisions may occur!
+            # See: https://groups.google.com/forum/#!topic/iterm2-discuss/0tO3xZ4Zlwg
+            local currentProfileName
+            currentProfileName=$(osascript -e 'tell application "iTerm2" to profile name of current session of current window')
```

Comment:
All variables and functions in neofetch follow this naming scheme `example_var_name`. Change the variable name to reflect this. `current_profile_name`